### PR TITLE
Stop playing audio when global player is closed

### LINKED ISSUE #775: Closing the global audio player does not stop audio playback
## Description

Closing the global audio player does not stop audio playback. When you close it, the audio continues to play.

## Reproduction

1. Go to `/search/audio?q=birds` with a small width viewport and play an audio
2. In the global player, click the `x` to close the global player
3. Note the audio continues to play

## Expectation

The audio should stop playing.

At least I think so. It seems counter-intuitive to me that closing the global audio player _doesn't_ stop the playback.

## Additional context

This should be a matter of calling `activeAudio.value.pause()` in the click handler for the close button.

Found while testing https://github.com/WordPress/openverse-frontend/pull/773

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### LINKED ISSUE #1157: Unsafe `route` access in `VGlobalAudioTrack`
## Description

https://sentry.io/share/issue/5746823abe77499b86dbdb8fe6330e1b/

## Reproduction

No idea how to reproduce.

This should be fixable with some null checks. Can be done in tandem with #1156 in a fully audit of unsafe route access in the app.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #1157 by @sarayourfriend
Fixes #775 by @sarayourfriend 

This PR branch is based on the `ts_vwaveform` branch (#1311), so should be merged after that PR.

## Description

The main fix of this PR is to set `activeAudio` obj value to undefined when the `Close` button is clicked on the Global Audio Player, in addition to ejecting active media item in the active media store. This ensures that the audio playback is stopped when the player is closed.

Other changes include converting the `VGlobalAudioSection` to TS, and simplifying `VGlobalAudioTrack` by removing the unnecessary layout computations and size prop (this can be re-added should we decide to use the global audio player on layouts other than mobile.

## Testing Instructions

Search for audio in the Audio view on mobile screen (http://localhost:8443/search/audio?q=cat), and play some audios from it. Close the audio player - the audio should stop.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VAudioTrack/VGlobalAudioTrack.vue [hunk 1]
```diff
@@ -4,14 +4,14 @@
     :aria-label="$t('audio-track.aria-label').toString()"
     role="region"
   >
-    <Component :is="layoutComponent" :audio="audio" :size="size">
+    <VGlobalLayout :audio="audio" size="m">
```
Comment (sarayourfriend) [inline src/components/VAudioTrack/VGlobalAudioTrack.vue:7]: Nice clean up!

**Review by sarayourfriend — APPROVED** [review]
LGTM! Tests well locally. I'm surprised we don't have to call `pause` on the active audio object. Glad this works :grin:

**Comment by obulat** [comment]
> I'm surprised we don't have to call `pause` on the active audio object.

I now wonder if I should add it in now :)

**Comment by sarayourfriend** [comment]
> I now wonder if I should add it in now :)

I tried it locally and it didn't seem to make a difference :shrug: Couldn't hurt though!

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**